components/molecules/Card.js

```diff
@@ -23,9 +23,20 @@ export default function Card(props) {
         return `${readingTime} min ${props.t["read"]}`;
     }
 
+    function selectCheck(obj){
+        let selection = window.getSelection();
+        if(selection.toString().length > 0){
+            obj.preventDefault();
+            return false;
+        }else{
+            return true;
+        }
```

Comment:
Juste le formatting, dis le moi si tu me trouve trop picky 😅 
```suggestion
        if (selection.toString().length > 0) {
            obj.preventDefault();
            return false;
        } else {
            return true;
        }
```

Reply:
faudrait vraiment que je regarde mon autoformatter, toute était setté avant de changer d'ordi lol